build.sbt

```
@@ -4879,6 +4888,33 @@ lazy val cleanPolyglotRoot = taskKey[Unit](
   "Helper task that prepares polyglot directory of a stdlib component"
 )
 
+lazy val `opencv-thin` = project
+  .in(file("lib/java/opencv-thin"))
+  .enablePlugins(JarExtractPlugin)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.openpnp" % "opencv" % opencvVersion
+    ),
+    inputJar := "org.openpnp" % "opencv" % opencvVersion,
+    jarExtractor := JarExtractor(
+      "nu/pattern/opencv/linux/x86_64/*.so" -> CopyToExtractDir(
+        Some(LinuxX86_64)
+      ),
+      "nu/pattern/opencv/osx/ARMv8/*.dylib" -> CopyToExtractDir(
```

- I don't like the name `CopyToExtractDir`
- `Some(...)` is (needlessly) repeated again and again
- why not just:

```suggestion
      "nu/pattern/opencv/osx/ARMv8/*.dylib" -> PolyglotLib(MacOSArm64)
```

---

Good idea. Done in 91e13c88c76d1b3330c10025adbee4e9119e4b81